app/javascript/stylesheets/application.scss

```diff
@@ -11,3 +11,80 @@ $govuk-assets-path: "~govuk-frontend/govuk/assets/";
 .govuk-template {
   background-color: white;
 }
+
+.thumbnail-left {
+  float:left;
+  padding-bottom: 5%;
+  width: 40%;
+  margin-left: 2%;
+}
+
+.thumbnail-right {
+   float:right;
+   padding-bottom: 5%;
+   width: 40%;
+   margin-right: 10%;
+ }
+
+.flash-archive {
+  border: 5px solid govuk-colour("green");
+  @include govuk-font($size: 19);
+  color: govuk-colour("black");
+  display: block;
+  overflow: hidden;
+  margin-bottom: govuk-spacing(6);
+  padding: govuk-spacing(2);
+}
+
+.moj-banner {
+  border-color: green;
+  color: govuk-colour("green");
+  font-size: 0; // Removes white space when using inline-block on child element.
+  margin-bottom: govuk-spacing(6);
+  padding: govuk-spacing(2);
+}
+
+
+.moj-banner__icon {
+  fill: currentColor;
+  float: left;
+  margin-right: govuk-spacing(2);
+}
+
+.moj-banner__message {
+  @include govuk-font($size: 19);
+  color: govuk-colour("black");
+  display: block;
+  overflow: hidden;
+}
+
+.moj-banner__message h2 {
+  margin-bottom: govuk-spacing(2);
+}
+
+
+.moj-banner__message h2:last-child,
+.moj-banner__message p:last-child {
+  margin-bottom: 0;
+}
+
+
+.moj-banner__assistive {
+  @include govuk-visually-hidden;
+}
+
+
+/* Style variants
+   ========================================================================== */
+
+.moj-banner--success {
+  border-color: govuk-colour("green");
+  color: govuk-colour("green");
+}
+
+
+.moj-banner--warning {
+  border-color: govuk-colour("red");
+  color: govuk-colour("red");
+}
+
```

Comment:
Have a look the [flashes component][1] I think some of them if not all of the are covered there 😄 

[1]: https://github.com/unboxed/bops/blob/master/app/javascript/stylesheets/components/_flashes.scss